### PR TITLE
LPS-177962 [Raylife AP]  Create Claim Details Page

### PR DESCRIPTION
**Pull request by joanagrego**
Related with --> [LPS-177962](https://issues.liferay.com/browse/LPS-177962)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+	return (
+		<div className="claim-details-container">
+			{!isClaimSettled && (
+				<div className="align-items-center bg-neutral-0 d-flex justify-content-center multi-steps-content">
+					<MultiSteps steps={steps.filter((step) => step.show)} />
+				</div>
+			)}
+
+			<div className="claim-detail-content d-flex py-4 row">
+				<div className="col-12 col-lg-12 col-md-12 col-sm-12 col-xl-3 d-flex mb-4">
```
Comment (claricebrasil) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx:195]: Are all these classes really necessary? Bc we're using 3 types of columns here, right? Large, middle and small

Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx:195]: You're right, they're not necessary.

**Review by claricebrasil — APPROVED** [review]
Great job!

**Review by eduardo-diniz — APPROVED** [review]
Good Job!!!

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+			<div className="claim-detail-content d-flex py-4 row">
+				<div className="col-xl-3 d-flex mb-4">
+					{!isClaimSettled && (
+						<Summary dataSummary={summaryClaimData} />
+					)}
+
+					{isClaimSettled && (
+						<Summary dataSummary={summaryClaimDataSettled} />
+					)}
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx]: Nice!

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:forward

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/pt-liferay-solutions/liferay-portal/pull/3485)

**Pull request closed without merging**